### ISSUE TITLE
Instance / Boss: Shadow Labyrinth / Blackheart the Inciter 

### ISSUE THREAD

**Issue by Sagrath** [dalaranwow/dalaran-wow#482]
Blackheart the Inciter (http://old.wowhead.com/npc=18667)
each time he did his Incite he would kinda reset...

according to wowhead he should do:
Incite Chaos (http://old.wowhead.com/spell=33676) 
"Incites chaos in the minds of all enemies, causing them to randomly attack each other."
but judging by the icon, he actually did:
Incite Chaos (http://old.wowhead.com/spell=33684) 
"Drives the target into a chaotic rage, attacking anyone nearby."
that last one also has a charm aura, and that causes all to assist, and him loosing agro thus resetting.

**Comment by Dasher2417** [comment]
STILL an issue. The boss cannot be killed, nor passed.

**Comment by gotnoface** [comment]
Blocking all of these quests and their associated chains:
The Codex of Blood (http://www.wowhead.com/object=182947/the-codex-of-blood)
The Book of Fel Names (http://www.wowhead.com/quest=10649/the-book-of-fel-names)
Entry into Karazhan (http://www.wowhead.com/quest=9831/entry-into-karazhan)
The Lexicon Demonica (http://www.wowhead.com/quest=10666/the-lexicon-demonica)
Wanted: Murmur's Whisper (http://www.wowhead.com/quest=11375/wanted-murmurs-whisper)

The problamtic ability:
Incite Chaos: Full party mind control. The party will go free for all, and start randomly killing each other. He will not take part in this fight. Lasts approx 15 seconds. Dumps aggro. The first one is cast about 20 seconds in the fight, then again approximately every 40 seconds during the whole fight.

Issue 1: Since the whole party turns friendly towards him, unless there is a pet attacking him, the fight is reset and so is his health, voiding any damage done to him.
Issue 2: Party is not attcaking each other, instead they are turned into his "pets" and follow him for the duration of the spell.
Issue 3: He is capable of chain casting Incite Chaos without any cooldown between the casts.

This makes him unbeatable by any party that can't do 150k damage in 15 secs or so from the start of fight. Possibly beatable by a party full of powerfull 80lvl pets.

If making a script for such fight is a "waste of time" why not at least siable the skill until it's resolved? It's been year and a half as was noted before.

**Comment by n-ghost** [comment]
Just to throw five more cents, same thing with Pathaleon and Skyriss, although those two can be interrupted (not sure if it works as intended though, wowhead won't mention any MC interrupts on either).

Related to https://github.com/dalaranwow/dalaran-wow/issues/1451

**Comment by Braniacz** [comment]
Still bugged

**Comment by budthechud** [comment]
New issue with Blackheart:
A friend and I successfully downed him, Incite Chaos worked fine however it will not let us leave the room.  When we attempt to leave it runs us back to his corpse.  We tried to run back out to the entrance of his room and were drawn back to him again.  We waited for him to despawn and it continued to occur. We cannot exit the area around where he stands on his platform.

**Comment by alexuktc** [comment]
I have been able to solo him (not having 150k dps done before the skill) for close to a year. I would get MC'd, act like his pet and when it runs off he will attack me without his HP resetting. Once he dies I need to relog to remove the "pet" return. Other than that it is doable now. But I have not tried with a group, and especially not with a group of ~70.

**Comment by KongFis** [comment]
He is not skipable since the door behind him only opens if you kill him. Killing him is possible (solo), but you can't go any furthur in dungeon. Your character will simply go back to his corpse as if you were mind controlled. Please fix it or make him skipable, either way.

**Comment by Rushor** [comment]
Scripts/ShadowLabyrinth: Blackheart the Inciter - Disable Mind Controle Effect

* closes: https://github.com/dalaranwow/dalaran-wow/issues/482
* i removed the mind controle event from this boss. Once https://github.com/dalaranwow/dalaran-wow/issues/1451 was fixed, we can enable that mind controle event for the boss again. But for now that disable should help the people to complete the instance (it's only a work around)
* commit: ab129e9c94d43b8edc86d0d1934fdb601f81f504

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.
